src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
The `.stars` class is fine as it is, but consider adding a comment to explain the purpose of this class for better maintainability.